libraries/postgres/src/util.js

```diff
@@ -9,7 +9,7 @@ exports.dollarQuote = str => {
     if (str.indexOf(quote) === -1) {
       return quote + str + quote;
     }
-    i = i + '-';
+    i = i + 'x';
```

Comment:
What's happening here? :)

Reply:
It turns out Postgres wants `$<tag>$`, and apparently `-` is not a valid character in the tag.  So I used `x` instead.  I noticed this when quoting a string that contained `$$`.